extensions/swift-package-index/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "swift-package-index",
+  "title": "Swift Package Index Search",
+  "description": "Search for Swift Packages via https://swiftpackageindex.com",
+  "icon": "command-icon.png",
+  "author": "maximkrouk",
+  "owner": "capturecontext",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Swift Packages",
```

Comment:
```suggestion
      "title": "Search Packages",
```

Reply:
I guess that putting the target into the title should help while searching extensions if you use extensions for both npm and spm for example, even if your suggestion looks a bit neater 🤔

I'm okay with the change, just wondering what you think. Maybe there is a way to add keywords? Don't remember it from the docs...

Reply:
Yes, you can also use the `keywords` property, we mention it in our docs: https://developers.raycast.com/information/manifest#command-properties

My proposed changed is to have this extension coherent with the rest of our extensions and commands. If you keep the command subtitle as the name of the extension, no need to repeat Swift in the command title.

Reply:
Am I right, that "tokens" from any title are used as keywords anyway and there is no need to manually add some?

Reply:
Yes, that is true as well